DirectProgramming/DPC++FPGA/Tutorials/Features/private_copies/README.md

```diff
@@ -0,0 +1,181 @@
+
+## Examining the Reports
+Locate `report.html` in the `private_copies_report.prj/reports/` or `private_copies_s10_pac_report.prj/reports/` directory. Open the report in any of Chrome*, Firefox*, Edge*, or Internet Explorer*.
+
+On the main report page, scroll down to the section titled "Estimated Resource Usage". Each kernel name ends in the `private_copies` attribute argument used for that kernel, e.g., `kernelCompute1` uses a `private_copies` attribute value of 1. You can verify that the number of RAMs used for each kernel increases with the private_copies value used, with the exception of private_copies 0, which instructs the compiler to choose a default value.
+
+## Running the Sample
+
+ 1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+     ```
+     ./private_copies.fpga_emu     (Linux)
+     private_copies.fpga_emu.exe   (Windows)
+     ```
+2. Run the sample on the FPGA device:
+     ```
+     ./private_copies.fpga         (Linux)
+     ```
+
+
+### Example of Output
+```
+PASSED_fpga_compile
+Kernel time when private_copies is set to 0: 1441.45 ms
+Kernel throughput when private_copies is set to 0: 0.568 GFlops
+Kernel time when private_copies is set to 1: 2916.440 ms
+Kernel throughput when private_copies is set to 1: 0.281 GFlops
+Kernel time when private_copies is set to 2: 1458.260 ms
+Kernel throughput when private_copies is set to 2: 0.562 GFlops
+Kernel time when private_copies is set to 3: 1441.450 ms
+Kernel throughput when private_copies is set to 3: 0.568 GFlops
+Kernel time when private_copies is set to 4: 1441.448 ms
+Kernel throughput when private_copies is set to 4: 0.568 GFlops
+PASSED: The results are correct
+```
+
+### Discussion of Results
+
+The stdout output shows the throughput (GFlops) for each kernel. 
+
+When run on the Intel® PAC with Intel Arria10® 10 GX FPGA hardware board, we see that the throughput doubles from using private_copies 1 to private_copies 2. Further increasing the value of private_copies does not increase the throughput achieved, i.e., increasing the private_copies above 2 will spend additional RAM resources for no additional throughput gain. As such, for this tutorial design, maximal throughput is best achieved by using private_copies 2. 
+
+Using private_copies 0 (or equivalently omitting the attribute entirely) also produced good throughput, indicating that the compiler's default heuristic chose a private_copies value of 2 or higher in this case.
+
+When run on the FPGA emulator, the private_copies attribute has no effect on runtime. You may notice that the emulator achieved higher throughput than the FPGA in this example. This is because this trivial example uses only a tiny fraction of the spatial compute resources available on the FPGA.
```

Comment:
```suggestion
When run on the FPGA emulator, the `private_copies` attribute has no effect on kernel time. You may actually notice that the emulator achieved higher throughput than the FPGA in this example. This is because this trivial example uses only a tiny fraction of the spatial compute resources available on the FPGA.
```